flag.go

```diff
@@ -490,6 +490,33 @@ func (f *StringFlag) ApplyWithError(set *flag.FlagSet) error {
 
 // Apply populates the flag given the flag set and environment
 // Ignores errors
+func (f *PathFlag) Apply(set *flag.FlagSet) {
+	f.ApplyWithError(set)
+}
+
+// ApplyWithError populates the flag given the flag set and environment
+func (f *PathFlag) ApplyWithError(set *flag.FlagSet) error {
```

Comment:
I take it we don't need to resolve the path in here since it'll be relative to the working directory.